include/uv.h

```diff
@@ -693,10 +693,22 @@ UV_EXTERN int uv_udp_send(uv_udp_send_t* req,
                           unsigned int nbufs,
                           const struct sockaddr* addr,
                           uv_udp_send_cb send_cb);
+UV_EXTERN int uv_udp_send_ex(uv_udp_send_t* req,
+                             uv_udp_t* handle,
+                             const uv_buf_t bufs[],
+                             unsigned int nbufs,
+                             const struct sockaddr* addr,
+                             socklen_t addrlen,
```

Comment:
I suggest using `unsigned int` instead of `socklen_t` because the latter is a POSIX-ism.

I'd normally suggest `size_t` but I'm fairly sure it's `unsigned int` on all platforms because of BSD legacy.